examples/spark/tabular-preset-automl.py

```diff
@@ -55,12 +55,13 @@ def main(spark: SparkSession, dataset_name: str, seed: int):
                 'mini_batch_size': 1000
             },
             linear_l2_params={'default_params': {'regParam': [1e-5]}},
-            reader_params={"cv": cv, "advanced_roles": False}
+            reader_params={"cv": cv, "advanced_roles": False},
+            computation_settings=("parallelism", 3)
         )
 
         oof_predictions = automl.fit_predict(
             train_data,
-            roles=roles,
+            roles=dataset.roles,
```

Comment:
the [same](https://github.com/sb-ai-lab/SLAMA/pull/7/files#r1223141108)

Reply:
These interfaces follows interfaces of LightAutoML, which is an "origin" of SLAMA.
Not sure if it is a good idea to change these intrefaces. Need a discussion on this matter. 
(though SLAMA change them if it is necessary for performance purposes or for convenience related to Spark specific)